backend/organization/utility/project.py

```diff
@@ -60,6 +70,45 @@ def create_new_project(data: Dict, source_language: Language) -> Project:
     project.save()
     return project
 
+<<<<<<< HEAD
+def add_project_member(project,user,user_role,role_in_project,availability):
+    """
+    Adds a user to a project. Assumes valid data at input.
+
+
+    """
+
+    ProjectMember.objects.create(
+                        project=project
+                        ,user=user
+                        ,role=user_role
+                        ,role_in_project=role_in_project
+                        ,availability=availability
+                    )
+    return 
+
+
+def get_project_admin_creators(project,limit_to_admins=False):
+    """
+    Returns a given project UserProfiles of Creators or Administrators. if limit_to_admins is set to True, only admins will be returned.
+    :param project: target project 
+    :type project: Project 
+    :param limit_to_admins: limit output to admins only 
+    :ype limit_to_amins: bool
+    """
+    targets_roles = Role.objects.filter(Q(name="Creator") | Q(name="Administrator")).all()
```

Comment:
I wouldn't go by name here. You can check whether `role_type__in=[Role.READ_WRITE_TYPE, Role.ALL_TYPE]`. 
Relying on name for roles can lead to some unexpected results :D